langchain/src/vectorstores/vectara.ts

```diff
@@ -0,0 +1,250 @@
+import { Document } from "../document.js";
+import { Embeddings } from "../embeddings/base.js";
+import { FakeEmbeddings } from "../embeddings/fake.js";
+import { getEnvironmentVariable } from "../util/env.js";
+import { VectorStore } from "./base.js";
+
+export interface VectaraLibArgs {
+  customer_id: number;
```

Comment:
Changed this to camel case for consistency in #1785

Reply:
sgtm